src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -197,49 +329,106 @@ def get_trough_and_peak_idx(template):
     """
     Return the indices into the input template of the detected trough
     (minimum of template) and peak (maximum of template, after trough).
-    Assumes negative trough and positive peak
+    Assumes negative trough and positive peak.
+
+    Parameters
+    ----------
+    template: numpy.ndarray
+        The 1D template waveform
+
+    Returns
+    -------
+    trough_idx: int
+        The index of the trough
+    peak_idx: int
```

Comment:
For example if we call this crest instead then we don't worry about the fact that for many extracellular recordings we want to define the peak as the most negative voltage?